src/wombats/game/finalizers.clj

```diff
@@ -11,19 +11,43 @@
                             [] meta)]
     (assoc cell :meta meta-update)))
 
-(defn- update-metadata-decay
-  [game-state]
+(defn- update-deterioration
+  "Determines the deterioration level and applys it to the element"
```

Comment:
typo `applies` 

I don't really think these things are a big deal, but I comment on them b/c this is going to be public facing eventually.